.changeset/healthy-kings-give.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-studio': patch
+---
+
+Add test for Association Viewer
```

Comment:
You don't need a message when you add test like this. You can delete this changeset as the other one should already suffice